src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1905,14 +1905,14 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		await this._processManager.setDimensions(rawXterm.cols, rawXterm.rows);
 	}
 
-	setShellType(shellType: TerminalShellType | undefined) {
+	setShellType(shellType: TerminalShellType | undefined) { // Why is shellType being shown as string here as well...
```

Comment:
Can remove all these string comments. I think your confusion is that when you debug it will show a string, that's because `TerminalShellType` is a set of const enums which are all strings.